encryption_works.md

```diff
@@ -339,53 +339,43 @@ Snowden was right to be cautious and to insist that he check Gellman's new PGP f
 
 ## Tails: The Amnesic Incognito Live System
 
-Using "properly implemented strong crypto systems" has a huge learning curve and requires dedicated users who are willing to put in extra work to take control of their own privacy, which is the main reason why OTR and PGP are not currently in widespread use. But even when you use these tools, how can you ensure "endpoint security" when you can't necessarily trust your operating system or other software that you depend on every day?
+Using "properly implemented strong crypto systems" has a steep learning curve. It required dedicated users to put in extra work to take control of their own privacy -- this is the main reason OTR and PGP are not used widely. But even when you use these tools, how can you ensure "endpoint security" when you can't necessarily trust your operating system and other software you depend on every day? 
 
-The solution is to use an entirely different operating system comprised completely of "software you can trust" when you have a serious need for real privacy. [Tails](https://tails.boum.org/about/index.en.html) helps solve this problem.
+The solution is to use an entirely different operating system comprised entirely of "software you can trust," meaning that you don't have to rely on other closed software. [Tails](https://tails.boum.org/about/index.en.html) helps solve this problem:
 
 > Tails is a live system that aims at preserving your privacy and anonymity. It helps you to use the Internet anonymously almost anywhere you go and on any computer but leave no trace using unless you ask it explicitly.
```

Comment:
I would put lines 346 - 351 back in before you get to the part where you describe how Tails is more difficult. That way people know what they are getting as a positive before we get to the negatives.